samples/sensor/gesture_polling/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Paul Timke <ptimkec@live.com>
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(gesture_polling)
```

Comment:
the way the sample is named and placed in the tree makes it sound it's generic but it's really not, it should be named after the specific sensor it tests